prototypes/notebook-generator/helper.py

```diff
@@ -0,0 +1,58 @@
+from string import Template
+
+
+class GeneratorHelper:
+
+    short_name = None
+    api = None
+
+    def __init__(self, params):
+        self.params = params
+
+    # common in here and code_template
```

Comment:
This function is repeated, what would be the practice.

Reply:
One possibility would be to make `template` a class instead of a list of strings, and put `format` on it as a method. Then it will be available everywhere you have a template instance.

Or you could extract it as a module-level function you import from wherever you need it